app/migrations/0008_movie_simplifiedtitle.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2016-11-16 18:13
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('app', '0007_movie_recentvisits'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='movie',
+            name='SimplifiedTitle',
+            field=models.CharField(blank=True, max_length=128),
```

Comment:
Why save the full title and a shortened version? I currently have code set up to simplify the movie title right before using it in the Twitter search in my local dev branch. Open to argument but I think simplifying right before use is a better solution, less stuff in the DB.